app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/HorizontalSnapScrollView.kt

```diff
@@ -0,0 +1,267 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.GestureDetector
+import android.view.GestureDetector.SimpleOnGestureListener
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
+import android.widget.HorizontalScrollView
+
+import androidx.annotation.IntRange
+import androidx.annotation.VisibleForTesting
+import androidx.core.view.children
+import androidx.core.view.forEach
+import androidx.core.view.updateLayoutParams
+
+import info.metadude.android.eventfahrplan.commons.logging.Logging.Companion.get
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.schedule.HorizontalSnapScrollState.Companion.FLING_COLUMN_MULTIPLIER
+import nerd.tuxmobil.fahrplan.congress.schedule.HorizontalSnapScrollState.Companion.SWIPE_THRESHOLD_VELOCITY
+import kotlin.math.ceil
+import kotlin.math.floor
+import kotlin.math.roundToInt
+
+import androidx.core.view.get
+
+class HorizontalSnapScrollView(context: Context?, attrs: AttributeSet?) :
```

Comment:
Please reorder the class content:

1. `companion object`
2. Properties
3. `init {}` block